language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/signature/SignatureHelpUtil.java

```diff
@@ -100,7 +100,8 @@ private SignatureHelpUtil() {
         labelBuilder.append("(");
         // Join the function parameters to generate the function's signature
         List<ParameterInfoModel> parameterInfoModels = signatureInfoModel.getParameterInfoModels();
-        for (ParameterInfoModel paramModel : parameterInfoModels) {
+        for (int i = 0; i < parameterInfoModels.size(); i++) {
+            ParameterInfoModel paramModel = parameterInfoModels.get(i);
             int labelOffset = labelBuilder.toString().length();
```

Comment:
You can just call `labelBuilder.length()` otherwise whole purpose of StringBuilder is lost since we convert toString per each cycle.